<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ddMainWindow</class>
 <widget class="QMainWindow" name="ddMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KALTAS</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="ExitBut">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>0</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>0</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>771</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tabCopy">
     <attribute name="title">
      <string>Copying</string>
     </attribute>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>711</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxSrc">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelSrc">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Диски и устройства</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditdest">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="destDialog">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>40</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>D</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelDest">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>20</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Файл назначения</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ddStartButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>70</y>
        <width>83</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Start </string>
      </property>
     </widget>
     <widget class="QPushButton" name="ddStopButton">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>70</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop/Pause</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>711</width>
        <height>311</height>
       </rect>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="rowCount">
       <number>3</number>
      </property>
      <property name="columnCount">
       <number>4</number>
      </property>
      <row/>
      <row/>
      <row/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>190</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Найденные устройства</string>
      </property>
     </widget>
     <widget class="QPushButton" name="fillTable">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>190</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Обновить</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelInfo">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>481</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabMount">
     <attribute name="title">
      <string>Mounting</string>
     </attribute>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>711</width>
        <height>141</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="ButtonMountSrc">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>70</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Src</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ButtonMountDest">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>70</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Dest</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ButtonMount">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>70</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Mount</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditMountSrc">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>191</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditMountDest">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>70</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>171</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Образ для монтирования</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>50</y>
         <width>171</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Директория назначения</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>10</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Просмотр образов </string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tabMobile">
     <attribute name="title">
      <string>Mobile</string>
     </attribute>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>741</width>
        <height>181</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QComboBox" name="widComboMobileDevice">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>211</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Найденные устройства</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="widLineDestDirMobile">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>40</y>
         <width>261</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>20</y>
         <width>231</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Куда сохранять информацию</string>
       </property>
      </widget>
      <widget class="QPushButton" name="widButChooseDir">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>40</y>
         <width>41</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>D</string>
       </property>
      </widget>
      <widget class="QPushButton" name="widButStartMobileImage">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>80</y>
         <width>80</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
      <widget class="QPushButton" name="widButStopMobileImage">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>80</y>
         <width>80</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
      <widget class="QPushButton" name="widButUpdateMobileDevice">
       <property name="geometry">
        <rect>
         <x>69</x>
         <y>80</y>
         <width>101</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Update device</string>
       </property>
      </widget>
      <widget class="QLabel" name="widLabelSizeMobileInfo">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>130</y>
         <width>681</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QProgressBar" name="widProgressBarMobile">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>691</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
      <widget class="QLabel" name="widLabelServiceInfo">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>90</y>
         <width>341</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>20</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Копирование с мобильных устройств</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>520</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Version 08072015</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>782</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>КАЛТАС</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <signal>changeVal(int,off_t)</signal>
  <slot>slotFillTable()</slot>
  <slot>SaveDestFile()</slot>
  <slot>ExitApp()</slot>
  <slot>ddStop()</slot>
  <slot>slotChangeVal(int,off_t)</slot>
  <slot>updateDisks()</slot>
 </slots>
</ui>
